.github/CONTRIBUTOR_GUIDELINES.md

```diff
@@ -0,0 +1,110 @@
+  * [Git Commit Message Guidelines](#git-commit-message-guidelines)
+  * [Fetching the Sources From GitHub](#fetching-the-sources-from-github)
+  * [Linting and Testing OpenCollar with Your Changes](#Linting-and-Testing-OpenCollar-with-Your-Changes)
+
+## Asking Support Questions
+
+We have active [discussion channels](https://discord.gg/WUmBMemYqf) where users and developers can ask questions.
+Please don't use the GitHub issue tracker to ask general questions.
+
+## Reporting Issues
+
+If you believe you have found a defect in OpenCollar or its documentation, use
+the GitHub issue tracker to report
+the problem to the maintainers. When reporting the issue, please provide the version of OpenCollar in use (Help > About).
+
+- [OpenCollar Issues · OpenCollarTeam/OpenCollar](https://github.com/OpenCollarTeam/OpenCollar/issues)
+- [OpenCollar Documentation · OpenCollarTeam/opencollarteam.github.io](https://github.com/OpenCollarTeam/opencollarteam.github.io)
+
+## Code Contribution
+
+As OpenCollar's user base continues to grow, any new functionality must:
+
+* be useful to many.
+* fit naturally into _what OpenCollar does best._
+* strive not to break existing functionality
+* close or update an open [OpenCollar Issue](https://github.com/OpenCollarTeam/OpenCollar/issues)
+
+If the feature is of some complexity, the contributor is expected to maintain and support the new feature in the future (answer questions in-world or in discord, fix any bugs etc.).
+
+It is recommended to open up a discussion on the [Issues Tab](https://github.com/OpenCollarTeam/OpenCollar/issues) to get feedback on your idea before you begin. 
+
+Any non-trivial code change needs to update an open [issue](https://github.com/OpenCollarTeam/OpenCollar/issues). A non-trivial code change without an issue reference with one of the labels `bug` or `enhancement` will not be merged.
+
+**Bug fixes are, of course, always welcome.**
+
+## Submitting Patches
+
+The [OpenCollar](https://github.com/OpenCollarTeam/OpenCollar) Team welcomes all contributors and contributions regardless of skill or experience level. If you are interested in helping with the project, we will help you with your contribution.
+
+### Code Contribution Guidelines
+
+Because we want to create the best possible product for our users and the best contribution experience for our developers, we have a set of guidelines which ensure that all contributions are acceptable. The guidelines are not intended as a filter or barrier to participation. If you are unfamiliar with the contribution process, the [OpenCollar](https://github.com/OpenCollarTeam/OpenCollar) team will help you and teach you how to bring your contribution in accordance with the guidelines.
+
+To make the contribution process as seamless as possible, we ask for the following:
+
+* Go ahead and fork the project and make your changes.  We encourage pull requests to allow for review and discussion of code changes.
```

Comment:
As i was mentioning in the discord. I very much am against any guidelines that do not leave open a path for contribution when there is not anyone to review.

Reply:
Please note that this guideline very much differentiates between (external) Contributors and Developers (Collaborator, Members of the Repository). These Guidelines are specifically for external contributors and shall not apply to the core development team present now.